openedx/adg/lms/webinars/urls.py

```diff
@@ -0,0 +1,10 @@
+"""
+All urls for webinars app
+"""
+from django.urls import path
+
+from .views import webinar_description_page_view
+
+urlpatterns = [
+    path('', webinar_description_page_view, name='webinar_event'),
```

Comment:
Shouldn't there be a slug or id to represent a particular webinar description page? 
Or is it a view for all webinars? In that case, we should make the name plural i.e `webinar_events`

Reply:
I have added a TODO to update it in its proper story. Actually, this URL and its view is not part of the story but were required and were created temporality